docs/testplan/dual_tor/dual_tor_muxcable_test_plan.md

```diff
@@ -0,0 +1,164 @@
+# Dual ToR MuxCable Test Plan
```

Comment:
Behavior has to be checked with a bad cable. Definition of bad cable is: 
1) One of the TOR ends of the Ycable is down (shutdown port or disconnect cable)
2) Keep the autoswitch disabled as that is how it will be in production

Tests
- Toggle the Mux : There should be 100% traffic loss
- Manual switch counter should increment by one (i.e. cable should not toggle back)

Above test is necessary to verify that the cable will act strictly as Layer1 and has no intelligence of its own once the auto-switch is disabled